src/validators/issues.rs

```diff
@@ -1,57 +1,105 @@
+//! A module for issues creation.
+
+/// Represents the severity of an [`Issue`].
+///
+/// [`Issue`]: struct.Issue.html
 #[derive(Serialize, Debug, Eq, PartialEq)]
 pub enum Severity {
+    /// A critical error
     Fatal,
+    /// Error
     Error,
+    /// Warning
```

Comment:
It is not a specification error, but there is probably something wrong in the data